### ISSUE TITLE
366 of 365 days studied

### ISSUE THREAD

**Issue by dae** [ankitects/anki#2961]
https://forums.ankiweb.net/t/stats-incorrect-number-of-days-studied/40133

**Comment by itslinotlie** [comment]
Hi!

I'm new to open source development and I think this would be a great first issue to work on. Would it be possible to "reserve" this for the next couple of days?

Thanks (:

**Comment by dae** [comment]
Sure, please go ahead.

**Comment by itslinotlie** [comment]
I'm currently a bit swamped with school work and still have not figured out how to properly run the dev environment (my changes aren't being reflected in `./run` ). If someone wants to take over by all means go for it, otherwise I'll come back to this in a bit.

**Comment by dae** [comment]
Reopened, as the fix appears to have caused two regressions:

- today is now in its own bin, and can barely be seen on the graph
- the oldest bin is only 4 days

Related report:

https://forums.ankiweb.net/t/anki-24-06-release-candidate/44926/34

**Comment by abdnh** [comment]
> today is now in its own bin, and can barely be seen on the graph

This is the case on the current stable, so it's not related?

> the oldest bin is only 4 days

Yes, this changed because the graph is no longer counting an additional day. The newest bin should include today to fix this.

**Comment by user1823** [comment]
> The newest bin should include today to fix this.

Makes sense. But, what would be the latest bin called then? Past 5 days?

**Comment by abdnh** [comment]
> But, what would the latest bin be called then? Past 5 days?

Will be called "0-4 days ago" by default.
